### PR TITLE
feat: Implement per-URL duration and refresh cycle configuration

### PR DESCRIPTION
**Pull request by parthsidpara**
This PR introduces a major enhancement to PiOSK, moving from a hardcoded display configuration for all URLs to a flexible, per-URL parameter system. 
Users can now define a specific display `duration` and a `refresh cycle` count for each individual URL in the dashboard.

image

Changes to duration or cycle values are highlighted until they are applied

image

### CODE REVIEW

File: scripts/switcher.sh [hunk 1]
```diff
@@ -1,24 +1,74 @@
...
+  # Increment the display counter for the current tab.
+  ((REFRESH_COUNTS[$CURRENT_TAB_INDEX]++))
+  echo "Tab $((CURRENT_TAB_INDEX + 1)): Display cycle ${REFRESH_COUNTS[$CURRENT_TAB_INDEX]} of $cycle_target."
+
+  # Check if it's time to refresh this specific tab.
+  if [ "${REFRESH_COUNTS[$CURRENT_TAB_INDEX]}" -ge "$cycle_target" ]; then
+    echo "Refreshing Tab $((CURRENT_TAB_INDEX + 1))."
+    
+    # Send Ctrl+r to refresh the current tab using wtype.
+    wtype -M ctrl r -m ctrl
```
[the remark this answers is not in the document]
Comment (parthsidpara) [reply scripts/switcher.sh:55]: The current `wtype` syntax is correct.
The `-M` flag presses the modifier, and the `-m` flag releases it. 
Without the `-m` ctrl flag, the `ctrl` key would remain pressed indefinitely, which could lead to unintended behavior.
Same goes for `wtype -M ctrl -P Tab -m ctrl`

**Review by debloper — APPROVED** [review]
Didn't check switcher.sh logic in-depth; but looks good on the surface. Let's merge this and test it on device for edge cases and bugs. Thanks for the PR.
